index.js

```diff
@@ -0,0 +1,130 @@
+const fs = require('fs');
+const os = require('os');
+const path = require('path');
+const util = require('util');
+
+const core = require('@actions/core');
+const toolCache = require('@actions/tool-cache');
+
+/** Get the expected file extension for the cr binary. */
+function getExecutableExtension() {
+  if (os.type().match(/^Win/)) {
+    return '.exe';
+  } else {
+    return '';
+  }
+}
+
+/** Get the download URL of the cr binary for the host running the action. */
+function getDownloadUrl(version) {
+  const downloadPattern = 'https://github.com/helm/chart-releaser/releases/download/v%s/chart-releaser_%s_%s';
+
+  switch (os.type()) {
```

Comment:
Same here; just a best practice type of thing or is this unnecessary since we're picking/defining the OS?